gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -106,6 +107,13 @@ export default function NxFormLayoutExample() {
         setFilePristine(false);
       };
 
+  const [multiFiles, setMultiFiles] = useState<FileList | null>(null),
```

Comment:
Seems like this should use the state helpers.  So should the single file upload for that matter, but worrying about that one is out of scope

Reply:
adopted state helpers for just the multiple example here: [6b6ffe6](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/6b6ffe606565a9ed63cfb808615cc64a7c23c4cf)